src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString.split(';').forEach((line) => {
+    const trimedLine = line.trim();
+
+    if (trimedLine) {
+      const [property, value] = trimedLine
+        .split(':')
+        .map((part) => part.trim());
+
+      styleObject[property] = value;
```

Comment:
There should be a check to ensure that both `property` and `value` are not empty strings before assigning them to `styleObject`. This will prevent adding entries with empty keys or values, which could lead to unexpected behavior.